ui/components/multichain/network-manager/network-manager.tsx

```diff
@@ -32,17 +31,20 @@ import { AddNetwork } from './components/add-network';
 import { NetworkTabs } from './network-tabs';
 import { useNetworkManagerState } from './hooks/useNetworkManagerState';
 
-// Router content component
-const NetworkManagerRouter = () => {
+export const NetworkManager = () => {
+  const dispatch = useDispatch();
   const t = useI18nContext();
-  const history = useHistory();
+  const navigate = useNavigate();
   const location = useLocation();
-  const dispatch = useDispatch();
 
   const { initialTab } = useNetworkManagerState();
 
+  const onClose = useCallback(() => {
+    dispatch(hideModal());
+  }, [dispatch]);
```

Comment:
Is it intentional to have both an `onClose` and a `handleClose` method in this component?  And also is the `useCallback` really helping?